SDK-naming.md

```diff
@@ -0,0 +1,30 @@
+# SDK package names
+
+## Problem Description
+
+Describe top level package names for distribution and documentation for packages
+Based on: https://groups.google.com/forum/#!topic/aerogear/pNmVTy0saDA
+
+## Expectations
+
+- Same prefixes standards will be used in the platforms.
+
+## Proposed Solution
+
+### Android:
+
+org.aerogear.ags.android.core - AeroGear Services Core
```

Comment:
shouldn't this be `org.aerogear.android.ags.core`? (ags and android swapped)

Reply:
My mistake. Swapping that now